verifier/src/qe_identity.rs

```diff
@@ -80,20 +142,68 @@ impl<'a> TryFrom<&SignedQeIdentity<'a>> for QeIdentity {
     }
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Clone, PartialEq)]
 #[serde(rename_all = "camelCase")]
-struct TcbLevel {
+pub struct TcbLevel {
     tcb: Tcb,
     tcb_date: String,
     tcb_status: AdvisoryStatus,
     #[serde(rename = "advisoryIDs", default)]
     advisory_ids: Vec<String>,
 }
 
-#[derive(Debug, Deserialize)]
-struct Tcb {
+impl TcbLevel {
+    pub fn new<'a, I, E>(tcb: Tcb, tcb_status: AdvisoryStatus, advisory_ids: I) -> Self
+    where
+        I: IntoIterator<Item = &'a E>,
+        E: ToString + 'a + ?Sized,
+    {
+        Self {
+            tcb,
+            tcb_date: UNIX_TIME_STR.to_string(),
```

Comment:
I wonder about this `tcb_date`, and some of the members in tcb.rs.
I added all the members in the json spec, but if we're not actually using them I wonder if we should just omit them.

Reply:
I think I'll leave for now. They're already here, they may help someone if they get a verification failure. Only time will tell if they are more of a bane thane a boon.